content/blog/how-to-use-dsc-logging.en.md

```diff
@@ -0,0 +1,118 @@
+---
+title: "How to Use Dsc Logging"
+date: 2019-09-24T16:37:42+02:00
+type: "post"
+draft: false
```

Comment:
Please add `author: authorhandle` metadata.
We'll use it when I get a chance to display a card of the author on the article.